webcomponents/tool/src/main/frontend/packages/sakai-grader/src/sakai-grader-rendering-mixin.js

```diff
@@ -53,7 +53,7 @@ export const graderRenderingMixin = Base => class extends Base {
                   ${this.groups ? html`
                   <div class="grader-groups">
                     <div>${this.i18n.group_label}</div>
-                    <sakai-group-picker .groups=${this.groups}
+                    <sakai-group-picker .groups=${this.groups.filter(group => group.users && group.users.length > 0)}
```

Comment:
is this doing the opposite of what the title of the PR says?

Reply:
@ern It could be, depending on how this is understood. For me, it's correct. The change proposes that the groups available in the group selector should only be those that have students (thereby filtering out the empty ones).

Reply:
@st-manu I'd put your groups filter into the data mixin, where the groups data is set. Less logic in the templates is usually better.